proto/osmosis/concentrated-liquidity/incentive_record.proto

```diff
@@ -10,26 +10,28 @@ option go_package = "github.com/osmosis-labs/osmosis/v14/x/concentrated-liquidit
 
 // IncentiveRecord represents an active perpetual incentive gauge for a pool
 message IncentiveRecord {
+  uint64 poolId = 1;
```

Comment:
Why do we have `poolId` as part of the record if the key in state includes pool id as well?

Reply:
We should be able to avoid duplicating state in both key and the value. It should be possible to extract these fields from the key by using:
https://github.com/osmosis-labs/osmosis/blob/316005aca4dcc3c01ca8249934b9907014ff55e8/osmoutils/store_helper.go#L110

Reply:
Discussed offline and came to a conclusion on a good approach tracked here: #4284